src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
   margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 200px);
```

Comment:
Two column should start at 488px. I think there should be one column for smaller screen.